tox.ini

```diff
@@ -4,39 +4,22 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py26, py27, pypy, pypy3.3-5.2-alpha1, py33, py34, py35, py36
+envlist = py27,
+          py34,
+          py35,
+          py36,
+          pypy,
+          pypy3,
+          lint
+skip_missing_interpreters = true
 
 [testenv]
-deps =
-    unittest2
-    pycurl>=7.19
-    requests>=0.8.8
-    mock
-    coverage
-commands =
-    python setup.py clean --all
-    python -W all -bb -W error::BytesWarning -m coverage.__main__ run setup.py test {posargs}
-setenv =
-    STRIPE_TEST_PYCURL = true
+description = run the unit tests under {basepython}
+commands = python setup.py test -a "{posargs:-n auto}"
 
-[testenv:py27]
-deps =
-    flake8
-    unittest2
-    pycurl>=7.19
-    requests>=0.8.8
-    mock
-    coverage
-commands =
-    flake8 stripe tests
-    python setup.py clean --all
-    python -W all -bb -W error::BytesWarning -m coverage.__main__ run setup.py test {posargs}
-
-[flake8]
-exclude =
-    six.py
-
-[coverage:run]
-source = stripe
-omit =
-    stripe/six.py
+[testenv:lint]
+description = run static analysis and style check using flake8
+basepython = python2.7
+deps = flake8
+commands = python -m flake8 --show-source stripe setup.py {posargs}
+           python -m flake8 --show-source tests {posargs}
```

Comment:
Could be a single command (moving/including `tests` to/with the first).

Reply:
👍